ansible/roles/dashboard-installer/scenarios/container.yml

```diff
@@ -28,6 +28,7 @@
       OPENSDS_AUTH_URL: "http://{{ host_ip }}/identity"
       OPENSDS_HOTPOT_URL: "http://{{ host_ip }}:50040"
       OPENSDS_GELATO_URL: "http://{{ host_ip }}:8089"
+      SODA_DELFIN_URL: "http://{{ host_ip }}:8190"
```

Comment:
wondering why these cannot be defined once and reused

Reply:
I think we have to look at this for the overall installer.
may be we can take it up in the next release when we re-look at the installer improvements.

Reply:
@kumarashit Logged an issue for this #431

Reply:
@anvithks this is new code line we are adding, i think we can make atleast this variable reusable. Other we can take as improvement as we go

Reply:
@kumarashit I made the SODA Delfin URL configurable from the `common.yml` file .
For the others we can track using the issue raised above.